tests/test_match_fft_norm.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from pyfar.classes.audio import match_fft_norm
+
+
+def test_input_fft_norm_type():
+    """Test assertion by passig non str type"""
+    with pytest.raises(TypeError,
+                       match='Parameter fft_norm_1 must be type str.'):
+        match_fft_norm(0, 'none')
+    with pytest.raises(TypeError,
+                       match='Parameter fft_norm_2 must be type str.'):
+        match_fft_norm('none', 0)
```

Comment:
Is not needed anymore :)

Reply:
You are totally right, I just forgot to look at the file containing the tests. Thaks for reminding me, I fixed this with my last commit.

One Question: It seems like travis is checking every commit I push, despite this is only a draft pull request. Is this behaviour what we want, as our monthly time on travis is limited?